docs/source/community/community-call-notes/2019-april.md

```diff
@@ -0,0 +1,121 @@
+# All-Jupyter Community Video Call - April 2019
+
+**Date:** 30 April 2019 at 9am PST (your [timezome](http://arewemeetingyet.com/Los%20Angeles/2019-04-30/09:00/Jupyter%20Community%20Call))
+
+**Video-conference link:** https://calpoly.zoom.us/my/jupyter
+
+
+## Welcome to the Team Meeting
+
+Hello!
+
+If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+Name / Institution / Github Handle
```

Comment:
Wondering if it would be better going forward to put the names at the bottom of the document? Less scrolling.

Reply:
That's a good idea. I moved the names to the bottom of each page under a sub-section titled "In Attendance" :)